### DISCUSSION TITLE
Projects page: Reconnecting moved or renamed projects (update process)

### DISCUSSION THREAD

**Discussion by diegoguerrad** [semaphoreci/semaphore#260] [Documentation improvements]
Our documentation currently has the following instructions: https://docs.semaphoreci.com/using-semaphore/projects#reconnecting-moved-or-renamed-projects

Based on previous cases, the existing process would be outdated. After a customer updates the name or URL of a repo, we (Semaphore) have to update it on our end for the customer. 

We could adjust the steps to have customers reach out through a support ticket with the following details:

- Previous GH repo URL
- New GH repo URL

Before adding any draft, is there any other info we'd need to update a repo URL?

**Comment by TomFern** [comment]
Created https://github.com/semaphoreci/semaphore/issues/266 to track the issue.

**Comment by diegoguerrad** [comment]
hi @TomFern instead of these [three bullet points](https://docs.semaphoreci.com/using-semaphore/projects#reconnecting-moved-or-renamed-projects:~:text=When%20this%20happens,Press%20Change) could we ONLY keep the instructions under the "INFO" section? 

Those 3 points would no longer be valid - instead customers have to reach out to us to have their repos updated.

**Comment by TomFern** [comment]
@diegoguerrad I removed the bullet points and left only the info section.

**Comment by diegoguerrad** [comment]
hi @TomFern thanks for the changes 🙇 but I'm still seeing this section. 
We should delete this part which is NO longer valid:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/66d90b1e-c9d6-48e8-84f9-9fbc2aedfab8" />

And replace it with this one as the official instructions:

<img width="500" alt="image" src="https://github.com/user-attachments/assets/6e7c401b-2665-4cb7-b7d3-c6095210d3ef" />

**Reply by TomFern** [reply]
Sorry, I forgot to push the latest version to prod. It should be fixed in a few hours from now.

**Comment by pieterocp** [comment]
Eh, tbh I'm only probably the only person on the planet to have moaned about this, might be one to just leave docs a bit broken until this is something the platform just does?

(Well, no idea how painful it is under actually fix it under the bonnet to handle renames, think this is closed source code,  but given the folk over at [github probably handle it gracefully enough](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) it shouldn't be light years out?)

(But then the dilemma of, do we document what we have, or what we want comes in, and really what we have today is what is used vs what we'd like. Day it's working, then yeah, docs update time 🚀 )
